archlinuxcn-mirrorlist

```diff
@@ -1,112 +1,128 @@
 ##
 ## Arch Linux CN community repository mirrorlist
-## Generated on 2023-08-25
+## Generated on 2023-11-06
 ##
 
-## Our main server (Amsterdam, the Netherlands) (ipv4, ipv6, http, https)
+## Our main server (Amsterdam, Netherlands) (ipv4, ipv6)
```

Comment:
Knowing which mirrors supports HTTP is helpful in case HTTPS doesn't work (e.g. certificate store problems, networking issues).